frontend/talentsearch/src/js/components/Home/partials/Featured/Featured.tsx

```diff
@@ -0,0 +1,144 @@
+import React from "react";
+import { useIntl } from "react-intl";
+
+import { imageUrl } from "@common/helpers/router";
+
+import Heading from "@common/components/Heading";
+import TALENTSEARCH_APP_DIR from "../../../../talentSearchConstants";
+
+import Block from "./Block";
+
+// Create the page component
+const Featured = () => {
+  const intl = useIntl();
+  // This array is just a temporary data object representing the content required by the feature blocks. This data will need to be migrated to wherever makes sense, and we'll also need dynamic routes and translated strings
+  const featured = [
+    {
+      key: "pride",
+      title: intl.formatMessage({
+        defaultMessage: "Pride in tech",
+        id: "muwFhL",
+        description: "Title for the pride in tech featured item",
+      }),
+      summary:
+        "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis. ",
```

Comment:
If you're going to leave lorem text in then lets make sure it's explicitly in scope for another issue on the board.  Could you add it to the AC for the "Learn More" issue or make a new issue and add it to the epic?

Reply:
Update AC for https://github.com/GCTC-NTGC/gc-digital-talent/issues/4024